src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -188,7 +188,7 @@ private CookieOptions BuildCookieOptions()
                         new ClaimsIdentity(
                             new[] { new Claim(SessionIdClaim, _sessionKey, ClaimValueTypes.String, Options.ClaimsIssuer) },
                             Options.AuthenticationScheme));
-                    ticket = new AuthenticationTicket(principal, null, Options.AuthenticationScheme);
+                    ticket = new AuthenticationTicket(principal, ticket.Properties, Options.AuthenticationScheme);
```

Comment:
we need to be more selective about the properties we keep in the cookie. only take what you need.